frontend/src/utils/hooks/useResponsiveWidth.tsx

```diff
@@ -7,23 +7,29 @@ export function useResponsiveWidth(
   // Initial spec state is set in useEffect when default geo is set
   const ref = useRef<HTMLDivElement>(document.createElement('div'))
 
+  const forceUpdate = () => {
+    setWidth((prevWidth) => prevWidth + 1)
+  }
+
```

Comment:
I don't think this is really doing anything is it? You're defining this function but then it doesn't call it anywhere. It's listed as a side effect dependency in `useEffect` array, but that just means "run this useEffect is the function `forceUpdate()` is redefined" which is impossible anyways since it's defined with a `const`. Did I miss anything here ?

Reply:
I'll make an issue for this for now.